docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -89,7 +98,7 @@ cert-manager-webhook-cf8f9f895-kkj5r       1/1     Running             0
 
 #### Create the root issuer
 
-```console
+```{"stage":"cert-manager", "multiline_EOF":true, "runtime":"bash", "label":"create root issuer"}
```

Comment:
nice default, only adding multiline_EOF as needed.

Reply:
I think this would be better named `"HereTag":"EOF"` since that's the name of that kind of tags.